src/Microsoft.SqlTools.ServiceLayer/DisasterRecovery/BackupUtilities.cs

```diff
@@ -136,6 +147,109 @@ public BackupConfigInfo GetBackupConfigInfo(string databaseName)
         }
 
         /// <summary>
+        /// Execute backup
+        /// </summary>
+        public void PerformBackup()
+        {
+            this.backup = new Backup();
+            this.SetBackupProps();
+            backup.Database = this.backupInfo.DatabaseName;
+            backup.Action = this.backupActionType;
+            backup.Incremental = this.IsBackupIncremental;
+            if (backup.Action == BackupActionType.Files)
+            {
+                IDictionaryEnumerator IEnum = this.backupInfo.SelectedFileGroup.GetEnumerator();
+                IEnum.Reset();
+                while (IEnum.MoveNext())
+                {
+                    string CurrentKey = Convert.ToString(IEnum.Key,
+                        System.Globalization.CultureInfo.InvariantCulture);
+                    string CurrentValue = Convert.ToString(IEnum.Value,
+                        System.Globalization.CultureInfo.InvariantCulture);
+                    if (CurrentKey.IndexOf(",", StringComparison.Ordinal) < 0)
+                    {
+                        // is a file group
+                        backup.DatabaseFileGroups.Add(CurrentValue);
+                    }
+                    else
+                    {
+                        // is a file
+                        int Idx = CurrentValue.IndexOf(".", StringComparison.Ordinal);
+                        CurrentValue = CurrentValue.Substring(Idx + 1, CurrentValue.Length - Idx - 1);
+                        backup.DatabaseFiles.Add(CurrentValue);
+                    }
+                }
+            }
+
+            bool bBackupToUrl = false;
```

Comment:
>bBackupToUrl [](start = 17, length = 12)

please don't use hungarian notation in variable names..so this should be backupToUrl